datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/auth/GetAccessTokenMetadataResolver.java

```diff
@@ -0,0 +1,51 @@
+package com.linkedin.datahub.graphql.resolvers.auth;
+
+import com.datahub.authentication.token.StatefulTokenService;
+import com.linkedin.access.token.DataHubAccessTokenInfo;
+import com.linkedin.datahub.graphql.authorization.AuthorizationUtils;
+import com.linkedin.datahub.graphql.exception.AuthorizationException;
+import com.linkedin.datahub.graphql.generated.AccessTokenMetadata;
+import com.linkedin.datahub.graphql.generated.EntityType;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.concurrent.CompletableFuture;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class GetAccessTokenMetadataResolver
```

Comment:
Please create a unit test for this new resolver.

Reply:
I'll add an API test. Would be better compared to unit test here

Reply:
API test added https://github.com/datahub-project/datahub/pull/9793/commits/79f179f028acafeac2fb3880ccfcb591da464c0f Please see if this satisfies the requirement. I will make sure the CI is green tomorrow.

Reply:
API tests added. Will ensure CI is green